SillyBusinessInc/SillyBusinessGame
# Feat/#114-loottable

## Purpose of this PR:
This PR is for the loottable class and config system. 

## How to test:
1. Pick any room
3. Assets -> create -> Configs/LootTable -> select some log actions and set up the table
3. Assets -> create -> Configs/RewardConfig -> select fall back loottable and loottable per room type 

Treasure Chest
1. Drag Treasure Chest Prefab in a room (any) 
2. Make sure your reward config is in the script 

### links: 
[Ticket](https://github.com/SillyBusinessInc/SillyBusinessGame/issues/113)
[Ticket2](https://github.com/SillyBusinessInc/SillyBusinessGame/issues/114)
